datamade/nmid-scrapers
# Scrape candidate offices

## Description

This PR adds a scraper to retrieve office information for candidates. It also reorganizes the scraper code and updates the Makefile to reflect the new file structure.

### Testing instructions

- Run `make data/processed/offices.csv` and confirm the appropriate file is created, and looks correct.
